platform/android/export/export_plugin.cpp

```diff
@@ -38,6 +38,7 @@
 #include "core/io/image_loader.h"
 #include "core/io/json.h"
 #include "core/io/marshalls.h"
+#include "core/config/project_settings.h"
```

Comment:
Includes should be ordered alphabetically (so `config` comes before `io`).

You can set up pre-commit hooks locally to get this fixed automatically for future PRs.
https://contributing.godotengine.org/en/latest/engine/guidelines/code_style.html#pre-commit-hook